### PR TITLE
Working branch

### PR DESCRIPTION
**Pull request by eliasecasta**
In this project we created a simple library where we can store books with all their information and if we have read them or not.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've both done well here.**  👌

However, let's make a few changes and we are in the clear.

- Please refer to comments on your code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: Js/main.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+/* eslint func-names: ["error", "never"] */
```
Comment (mikenath223) [inline Js/main.js]: - [x] Kindly remove this disabled rule.

File: Js/main.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+
+function displayAllBooks(books) {
+  books.forEach((book, index) => {
+    content += bookCard(book, index);
+  });
+}
+
+$('form').on('submit', function (event) {
+  event.preventDefault();
+  content = '';
```
Comment (mikenath223) [inline Js/main.js]: - [x] Also please name this function to avoid a linting error.

File: Js/main.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+    </div>
+`;
+}
+
+function displayAllBooks(books) {
+  books.forEach((book, index) => {
+    content += bookCard(book, index);
+  });
+}
+
```
Comment (mikenath223) [inline Js/main.js]: - [x] Please use es6 arrow functions for these and other similar functions.

File: Js/main.js (hunk 1 above)
Comment (amita-roy) [reply Js/main.js]: Removed  /* eslint func-names: ["error", "never"] */  comment as suggested

File: Js/main.js (hunk 2 above)
Comment (amita-roy) [reply Js/main.js]: Added named function instead of anonymous callback functions.

File: Js/main.js (hunk 3 above)
Comment (amita-roy) [reply Js/main.js]: Converted above functions to an arrow function syntax.

File: Js/main.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+let myLibrary = [];
+const addNewBookButton = document.querySelector('button.newBook');
+
+function Book(title, author, pages, id) {
+  this.id = id;
+  this.title = title;
+  this.author = author;
+  this.pages = pages;
+}
```
Comment (christianotieno) [inline Js/main.js]: ## Use es6 arrow syntax
Please convert this typed function to es6 arrow syntax.

File: Js/main.js [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+function bookInfo(book) {
+  return `${book.title}, by ${book.author}, ${book.pages} pages`;
+}
```
Comment (christianotieno) [inline Js/main.js]: ## Use es6 arrow syntax
Please convert this typed function to es6 arrow syntax.

File: Js/main.js [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+Book.prototype.info = bookInfo(this);
+
+Book.prototype.read = 'Not Read';
+
+function addBookToLibrary(title, author, pages, id) {
+  const newBook = new Book(title, author, pages, id);
+  myLibrary.push(newBook);
+}
```
Comment (christianotieno) [inline Js/main.js]: ## Use es6 arrow syntax
Please convert this typed function to es6 arrow syntax.

File: Js/main.js [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+  </div>
+`;
+
+const displayAllBooks = (books) => {
+  books.forEach((book, index) => {
+    content += bookCard(book, index);
+  });
+};
+
+function formSubmission(event) {
```
Comment (christianotieno) [inline Js/main.js]: ## Use es6 arrow syntax
Please convert this typed function to es6 arrow syntax.

File: Js/main.js [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+      const card = document.getElementById(`book-${index}`);
+      if (card) {
+        card.remove();
+      }
+    });
+
+    function readStatus() {
+      return $(this).html() === 'Read'
+        ? $(this).html('Not Read')
+        : $(this).html('Read');
```
Comment (christianotieno) [inline Js/main.js]: ## Use es6 arrow syntax
Please convert this typed function to es6 arrow syntax.

**Review by christianotieno — CHANGES_REQUESTED** [review]
## Changes Required ♻️ 
Hello @e71az  & @Amita-Roy 👋 
Thank you for working the above-suggested changes. We still have some changes that need your attention. I have some suggestions down below before we proceed. 👇 

### Happy coding 💻 

#### Christian Otieno (TSE)

File: Js/main.js (hunk 5 above)
Comment (amita-roy) [reply Js/main.js]: Converted

File: Js/main.js (hunk 6 above)
Comment (amita-roy) [reply Js/main.js]: Converted

File: Js/main.js (hunk 4 above)
Comment (amita-roy) [reply Js/main.js]: Changed it

File: Js/main.js (hunk 7 above)
Comment (amita-roy) [reply Js/main.js]: Converted

File: Js/main.js (hunk 8 above)
Comment (amita-roy) [reply Js/main.js]: Converted

**Review by Dandush03 — CHANGES_REQUESTED** [review]
# Status: :green_circle: Approved :rocket: 
## Great work, keep it up!
# Happy Coding :nerd_face: :computer:

**Pull request merged**